src/MedicalCard/detailedIcon.style.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+const DetailedIconStyles = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  i {
+    font-size: 26px;
+    color: #575756;
+  }
+
+  .icon-detail {
+    text-align: center;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 1;
+    color: #575756;
```

Comment:
i see a lot of colors inlined - should they be organized like a pallet in one place?

Reply:
So an inlined color would be, like 
```html
<div style={color: #23432} </div>
```

So I'm slightly confused but I'm going to interpret this as, why don't we have one location where we import the colors (like a palette, or a custom theming thing). And if that's the case then its a good point. Probably should be its own thing but Christos has also mentioned that before so might be an idea.